data/gc/monster_gc.lua

```diff
@@ -0,0 +1,15 @@
+function monster_gc_memory()
```

Comment:
This is useless. By storing Loot and MonsterSpell objects here you prevent the garbage collector from cleaning it up until this function is called.